$OpenBSD: patch-src_evolution-import-rss_c,v 1.3 2008/11/23 12:07:45 ajacoutot Exp $

Increase the time-out to give evolution a fair chance to launch.

--- src/evolution-import-rss.c.orig	Wed Oct 22 17:27:37 2008
+++ src/evolution-import-rss.c	Sun Nov 23 13:05:02 2008
@@ -176,7 +176,7 @@ main (int argc, char *argv[])
 	{
 		system("evolution&");
 		g_print("fireing evolution...\n");
-		sleep(30);
+		sleep(60);
         	send_dbus_ping ();
 		g_timeout_add (EVOLUTION_PING_TIMEOUT, err_evo_cb, (gpointer)i++);
 		g_main_loop_run(loop);
